build/ndm-operator/Dockerfile.in

```diff
@@ -0,0 +1,14 @@
+#
+# This Dockerfile builds node-disk-operator
+# 
+
+#Set the base image
+FROM @BASEIMAGE@
+
+ARG ARCH
+
+RUN apt update
+
+ADD ndo /usr/local/bin/ndo
+
+CMD ["./ndo"]
```

Comment:
IMO, `ENTRYPOINT` should be used instead of `CMD`. Even though both serve the same purpose, since we are using the docker image as wrapper for our ndo binary, we need `ndo`  to execute, even if user tries to override with a command.

Reply:
Fixed.